src/main/java/stream/flarebot/flarebot/commands/moderation/WelcomeCommand.java

```diff
@@ -183,29 +185,15 @@ public CommandType getType() {
     }
 
     private void sendWelcomeTable(List<String> messages, int page, TextChannel channel) {
-        int messagesLength = 15;
-        int pages =
-                messages.size() < messagesLength ? 1 : (messages.size() / messagesLength) + (messages.size() % messagesLength != 0 ? 1 : 0);
-        int start = messagesLength * (page - 1);
-        int end = Math.min(start + messagesLength, messages.size());
-        if (page > pages || page < 0) {
-            MessageUtils.sendErrorMessage("That page doesn't exist. Current page count: " + pages, channel);
-        } else {
-            List<String> messagesSub = messages.subList(start, end);
-            List<List<String>> body = new ArrayList<>();
-            int i = 0;
-            for (String messagesMessage : messagesSub) {
-                List<String> part = new ArrayList<>();
-                part.add(String.valueOf(i));
-                part.add(MessageUtils.escapeMarkdown(messagesMessage));
-                body.add(part);
-                i++;
-            }
-            List<String> header = new ArrayList<>();
-            header.add("Id");
-            header.add("Message");
-            channel.sendMessage(MessageUtils.makeAsciiTable(header, body, " Messages Page " + GeneralUtils
-                    .getPageOutOfTotal(page, messages, messagesLength))).queue();
+        List<String> messagesWithId = new ArrayList<>();
```

Comment:
What may be an idea for these:
Have a "page" per message. Also be able to change the "Page 1/2" stuff with "Message 1/2". Could be quite nice.